reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/brownfield-byo-bastion.json

```diff
@@ -0,0 +1,1029 @@
+          {
+            "InstancePort": { "Ref" : "MasterApiPort" },
+            "InstanceProtocol": "TCP",
+            "LoadBalancerPort": { "Ref" : "MasterApiPort" },
+            "Protocol": "TCP"
+          }
+        ],
+        "SecurityGroups": [{"Ref": "MasterExtElbSG"}],
+        "Subnets": [
+          {"Ref": "PublicSubnet1"},
+          {"Ref": "PublicSubnet2"},
+          {"Ref": "PublicSubnet3"}
+            ],
+        "Instances": [
+          {"Ref": "Master01"},
+          {"Ref": "Master02"},
+          {"Ref": "Master03"}
+            ]
+      }
+    },
+    "InfraElb": {
+      "Type": "AWS::ElasticLoadBalancing::LoadBalancer",
+      "Properties": {
+        "CrossZone": "true",
+        "Tags": [ {"Key": "Name", "Value": "ose_router_elb"} ],
+        "HealthCheck": {
+          "HealthyThreshold" : "2",
+          "Interval" : "5",
+          "Target" : "TCP:443",
+          "Timeout" : "2",
+          "UnhealthyThreshold" : "2"
+        },
+        "Listeners":[
+          {
+            "InstancePort": "443",
+            "InstanceProtocol": "TCP",
+            "LoadBalancerPort": "443",
+            "Protocol": "TCP"
+          },
+          {
+            "InstancePort": "80",
+            "InstanceProtocol": "TCP",
+            "LoadBalancerPort": "80",
+            "Protocol": "TCP"
+          }
+        ],
+        "SecurityGroups": [ { "Ref": "InfraElbSG" } ],
+        "Subnets": [
+          {"Ref": "PublicSubnet1"},
+          {"Ref": "PublicSubnet2"},
+          {"Ref": "PublicSubnet3"}
+        	],
+        "Instances": [
+          {"Ref": "InfraNode01"},
+          {"Ref": "InfraNode02"}
+            ]
+      }
+    },
+    "NodePolicy": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "AssumeRolePolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Principal": { "Service": [ "ec2.amazonaws.com" ] },
+              "Action": [ "sts:AssumeRole" ]
+            }
+          ]
+        },
+        "Policies": [
+          {
+            "PolicyName": "node-describe",
+            "PolicyDocument": {
+              "Version" : "2012-10-17",
+              "Statement": [
+                {
+                  "Effect": "Allow",
+                  "Action": "ec2:Describe*",
+                  "Resource": "*"
+                }
+              ]
+            }
+          },
+          {
+            "PolicyName": "node-attach",
+            "PolicyDocument": {
+              "Version" : "2012-10-17",
+              "Statement": [
+                {
+                  "Effect": "Allow",
+                  "Action": "ec2:AttachVolume",
+                  "Resource": "*"
+                }
+              ]
+            }
+          },
+          {
+            "PolicyName": "node-detach",
+            "PolicyDocument": {
+              "Version" : "2012-10-17",
+              "Statement": [
```

Comment:
I think all three of these statements can be combined under a single PolicyDocument. I think I copy and pasted from the upstream kube example and never consolidated it in demo-env-ansible.